docs/pages/versions/unversioned/sdk/updates.md

```diff
@@ -47,6 +47,10 @@ In development mode, or any other environment in which `expo-updates` is disable
 
 (_string_) The name of the release channel currently configured in this standalone or bare app.
 
+### `Updates.updateId`
+
+(_string_) If `expo-updates` is enabled, the UUID that uniquely identifies the currently running update. The UUID is represented in its canonical string form (`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`) and will always use lowercase letters. In development mode, or any other environment in which `expo-updates` is disabled, this string is empty.
```

Comment:
I lean towards making this a nullable string. Regardless of the types we use, this field's value can be either the update ID if there is one, or a value indicating there is no update ID. Either way an application should handle both cases, and by making the type nullable, we make it more obvious that the code needs to handle the case where the update ID might be missing instead of sometimes sending an empty string as the release ID to Sentry for example.